### PR TITLE
Add Quarkus CXF to the Platform

### PR DESCRIPTION
**Pull request by ppalaga**
Fix https://github.com/quarkiverse/quarkus-cxf/issues/877

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -354,6 +355,25 @@
                                         </test>
                                     </tests>
                                 </member>
+                                <member>
+                                    <name>CXF</name>
+                                    <bom>io.quarkiverse.cxf:quarkus-cxf-bom:${quarkus-cxf.version}</bom>
+                                    <release>
+                                        <lastDetectedBomUpdate>io.quarkus.platform:quarkus-cxf-bom:3.0.0.CR1</lastDetectedBomUpdate>
```
Comment (ppalaga) [inline pom.xml:362]: @aloubyansky what should lastDetectedBomUpdate actually be for a newly added member?

Comment (aloubyansky) [reply pom.xml:362]: It simply should not be configured.

Comment (ppalaga) [reply pom.xml:362]: How simple!

Comment (aloubyansky) [inline pom.xml:362]:
```suggestion
```

**Comment by ppalaga** [comment]
I am still in the process of fixing the failed tests both on side of Camel Quarkus and Quarkus CXF

**Comment by ppalaga** [comment]
d9a87e3: fixed the conflict in angus-mail version between QCXF and Camel Quarkus (see https://github.com/quarkiverse/quarkus-cxf/issues/899 )

**Comment by ppalaga** [comment]
95053b : added some workarounds for failing tests and checks

**Comment by ppalaga** [comment]
I am going to disable org.apache.camel.quarkus.component.cxf.soap.client.it.CxfSoapClientTest for now, because it is passing in the subsequent #851 with newer Camel.

**Comment by ppalaga** [comment]
Hopefully the CI is going to pass now. If not, feel free to do whatever needed to get this in bc. I am away till Tuesday. It would be nice to get this in, because the subsequent #851 stands in top of this one.

**Comment by ppalaga** [comment]
Let me try to resolve the conflicts

**Comment by aloubyansky** [comment]
I fixed this target dir issue upstream just in case.

**Pull request closed without merging**